modules/timeutil/since.go

```diff
@@ -117,7 +118,9 @@ func timeSincePro(then, now time.Time, lang translation.Locale) string {
 // TimeSince calculates the time interval and generate user-friendly string.
 func TimeSince(then time.Time, lang translation.Locale) template.HTML {
 	timestamp := then.Format(time.RFC3339)
-	return template.HTML(fmt.Sprintf(`<relative-time class="time-since" prefix="%s" datetime="%s">%s</relative-time>`, lang.Tr("packages.versions.on"), timestamp, timestamp))
+	return template.HTML(fmt.Sprintf(`<relative-time class="time-since" title="" data-tooltip-content="%s" data-tooltip-interactive="true" prefix="%s" datetime="%s">%s</relative-time>`,
```

Comment:
Do we need this empty `title` attribute? Also I think we need to somehow convince `<relative-time>` to rewrite `data-tooltip-content` for local timezone.

Reply:
If we don't `title=""` we'll have 2 tooltips on hover
![image](https://user-images.githubusercontent.com/20454870/230793631-04f4bfa5-7bbf-4b35-9752-4c2af53971ac.png)

Reply:
The hardcoded `title` attribute is all over the [module](https://github.com/github/relative-time-element/blob/244af669cdc426da9a3a4b1524b907844991ef15/src/relative-time-element.ts), so I think we should probably just let it set and maintain it and device a solution on top of it to set `data-tooltip-content`.

Reply:
Let's revert the server change and let the element set `title` as it does. We'll find a better solution to attach tippy in JS.